BookHub-frontend/src/containers/homePage/homePage.container.js

```diff
@@ -1,12 +1,14 @@
 import React, {Component} from 'react';
 import BooksListing from '../booksListing/booksListing.container';
+import RegisterPage from '../registerPage/registerPage.container';
 
 class HomePage extends Component {
 	render() {
 		return (
 			<div>
 				<h1>Home</h1>
 				<BooksListing/>
+				{/* <RegisterPage/> */}
```

Comment:
pls remove this, `RegisterPage` should only be rendered if user is on `/register`